README.md

```diff
@@ -212,5 +212,8 @@ found in `doc_deps.deb.txt`:
 To build the documentation, ensure that you are in the top-level
 `/ceph` directory, and execute the build script. For example:
 
-	admin/build-doc
+	./admin/build-doc
 
+The built documentation will be available at buid-doc/output. There you will have access to the man and html pages.
```

Comment:
@rs-fabrica @zdover23 i don't really like this change. as `README.md` is now cluttered. it's full of useful information. IMHO, the longer it is the less people is willing to read it. i think it should only provide essential information and redirect the readers for further reading if they are not satisfied. i think we should not use it as an alternative of `doc/index.rst` .

Reply:
AFAIU this README is trying to provide build instructions for several use cases. My suggestion is just completing these instructions for one use case.

This extra info I would like to include would have saved me quite some time as many times when I try to improve Ceph's documentation, I forget where my newly built doc is.

I believe I'm probably not the only person that has this issue.

I can't see why should suggestion should be compared to trying to be "an alternative of doc/index.rst". It really seems to me as a gross exageration even as a hyperbole.